### ISSUE TITLE
One line daily forecast

### ISSUE THREAD

**Issue by mattyrc** [chubin/wttr.in#447]
Are we currently able to output the next few days high forecast in one line?

I'm currently using the following one line to get the current temperature:
`curl wttr.in/Vegas,NV?format=1\&u`
`⛅️  +89°F`

I'd like to be able to see something like:
`⛅️  +89°F, ⛅️  +91°F, ⛅️  +87°F`

Or maybe a way to just simply just print the forecast for tomorrow?

Sorry if this is has been covered before. I've searched and read through the readme and couldn't find anything.

Thanks

**Comment by chubin** [comment]
Yes, it is a good idea + it would be great to have one-line plots,
like this
```
 ⢠⣄⡴⣸
```
 or this:
```
 ▅▁▁▇▇▂▂▁▁▁▂▂ 
```

**Comment by giteauserbit** [comment]
+1 here. That would be great to have a one-liner like @mattyrc suggested.
